package.json

```diff
@@ -33,7 +33,7 @@
     "url": "https://github.com/azu/espower-babel/issues"
   },
   "dependencies": {
-    "babel-core": "*",
```

Comment:
@lookfirst Thanks! all right.
I think that `"babel-core": "^4.0.0"` is better because [Babel](https://github.com/babel "Babel") adopt [semver](https://github.com/npm/node-semver "semver").
Other [babel plugins](https://github.com/babel "Babel") use "^4.0.0".

So what do you think?